common/src/main/java/com/wynntils/models/containers/LootChestModel.java

```diff
@@ -37,16 +38,21 @@ public int getDryCount() {
         return dryCount.get();
     }
 
-    public Integer getDryBoxes() {
+    public int getDryBoxes() {
         return dryBoxes.get();
     }
 
+    public int getChestCount() {
```

Comment:
Maybe name it getOpenedChestCount() instead?

Reply:
Done